content/en/blog/_posts/2025-xx-xx-jobs-managedby-goes-ga.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: blog
+title: "Kubernetes v1.35: Job Managed By Goes GA"
+date: 2025-XX-XX
+draft: true
+slug: kubernetes-v1-35-job-managedby-for-jobs-goes-ga
+author: >
+  [Dejan Zele Pejchev](https://github.com/dejanzele) (G-Research),
+  [Michał Woźniak](https://github.com/mimowo) (Google)
+---
+
+In Kubernetes v1.35, the `managedBy` field for Jobs has reached general availability (GA).  
+This post gives a tour of what `managedBy` does, why it was added, and how it is used today, most notably by
+[MultiKueue](https://kueue.sigs.k8s.io/docs/concepts/multikueue/) for multi-cluster Job dispatching.
```

Comment:
While this is ok, I'm thinking we can make it a bit more attractive.

Maybe:

---

In Kubernetes v1.35, the managedBy field for Jobs graduated to General Availability (GA).

This feature allows external controllers to take full responsibility for Job reconciliation, unlocking powerful scheduling patterns like multi-cluster dispatching with [MultiKueue](https://kueue.sigs.k8s.io/docs/concepts/multikueue/).